apps/callscreen/test/unit/call_screen_test.js

```diff
@@ -912,6 +906,7 @@ suite('call screen', function() {
     var timeNode;
     setup(function() {
       this.sinon.useFakeTimers();
+      this.sinon.stub(navigator.mozL10n, 'ready', function(cb) { cb(); });
```

Comment:
I just realized I don't need that anymore as I've removed the last reference to mozL10n.ready() from the code